src/app/Topology/Shared/Entity/EntityDetails.tsx

```diff
@@ -214,24 +229,110 @@ export const TargetDetails: React.FC<{
     ];
   }, [serviceRef]);
 
+  const onToggle = React.useCallback(() => setExpanded((v) => !v), [setExpanded]);
+
   return (
     <DescriptionList {...props} columnModifier={columnModifier}>
-      {_transformedData.map((d) => (
-        <DescriptionListGroup key={d.key}>
-          <DescriptionListTermHelpText>
-            <Popover headerContent={d.helperTitle} bodyContent={d.helperDescription}>
-              <DescriptionListTermHelpTextButton>{d.title}</DescriptionListTermHelpTextButton>
-            </Popover>
-          </DescriptionListTermHelpText>
-          <DescriptionListDescription style={{ userSelect: 'text', cursor: 'text' }}>
-            {d.content}
-          </DescriptionListDescription>
-        </DescriptionListGroup>
-      ))}
+      {_transformedData.map(mapSection)}
+      <ExpandableSection
+        toggleText={isExpanded ? 'Show less' : 'Show more'}
+        onToggle={onToggle}
+        isExpanded={isExpanded}
+      >
+        <MBeanDetails isExpanded={isExpanded} connectUrl={serviceRef.connectUrl} />
+      </ExpandableSection>
     </DescriptionList>
   );
 };
 
+const MBeanDetails: React.FC<{ isExpanded: boolean; connectUrl: string }> = ({ isExpanded, connectUrl }) => {
+  const context = React.useContext(ServiceContext);
+  const [dayjs, dateTimeFormat] = useDayjs();
+  const addSubscription = useSubscriptions();
+  const [mbeanMetrics, setMbeanMetrics] = React.useState({} as MBeanMetrics);
+
+  React.useEffect(() => {
+    if (isExpanded) {
+      addSubscription(
+        context.api
+          .graphql<MBeanMetricsResponse>(
+            `
+            query MBeanMXMetricsForTarget($connectUrl: String) {
+              targetNodes(filter: { name: $connectUrl }) {
+                mbeanMetrics {
+                  runtime {
+                    startTime
+                    vmVendor
+                    vmVersion
+                  }
+                  os {
+                    version
+                    arch
+                    availableProcessors
+                  }
+                }
+              }
+            }`,
+            { connectUrl }
+          )
+          .pipe(map((resp) => resp.data.targetNodes[0].mbeanMetrics))
+          .subscribe(setMbeanMetrics)
+      );
+    }
+  }, [isExpanded, addSubscription, connectUrl, context.api, setMbeanMetrics]);
+
+  const _collapsedData = React.useMemo(() => {
+    return [
+      {
+        key: 'Start Time',
+        title: 'Start Time',
+        helperTitle: 'Start Time',
+        helperDescription: 'The time when this JVM process started.',
+        content: dayjs(mbeanMetrics?.runtime?.startTime || 0)
+          .tz(dateTimeFormat.timeZone.full)
+          .format('LLLL'),
+      },
+      {
+        key: 'JVM Version',
+        title: 'JVM Version',
+        helperTitle: 'JVM Version',
+        helperDescription: 'The version of the JVM.',
+        content: mbeanMetrics.runtime?.vmVersion,
```

Comment:
Looks like the content of these configurations can be undefined or null. We should show some indication instead of blank space. For example, 

```ts
 <EmptyText text="No JVM ID" />
```